src/scss/components/optional/_ratings.scss

```diff
@@ -18,9 +21,11 @@
     $star-size--medium          : 28px;
     $star-size--large           : 42px;
 
+  // https://github.com/justeat/f-icons/blob/master/icons/star-filled.svg
+  @warn 'To use the rating component you will need to add references to the SVGs in f-icons';
 
     .c-rating {
-        @extend %c-icon--star--empty !optional;
+        // @extend %c-icon--star--empty !optional;
```

Comment:
Should these comments still be here?

Reply:
Added a comment in for the reason. It's mainly so people can port over from the old icon lib.